app/controllers/web/dashboard/test_task_assignments_controller.rb

```diff
@@ -14,7 +14,7 @@ def index
       def show; end
 
       def activate
-        Ops::Developer::Activate.call(user: @candidate)
+        Ops::Developer::Activate.call(user: @candidate, approver: current_user.id)
```

Comment:
let's call parameter `as` instead of `approver`.
with such approach, we'll be able to add such parameter to app operations, so it'll be a kinda universal.